xwayland/server.c

```diff
@@ -232,8 +232,6 @@ static int xserver_handle_ready(int signal_number, void *data) {
 		.wm_fd = server->wm_fd[0],
 	};
 	wlr_signal_emit_safe(&server->events.ready, &event);
-	/* ready is a one-shot signal, fire and forget */
-	wl_signal_init(&server->events.ready);
```

Comment:
There's another place where this is done:

https://github.com/swaywm/wlroots/blob/c93016028680a0ce2bac2f2b8eeff5e5c1ea467e/xwayland/xwayland.c#L58

We should probably remove this one as well.